libs/features/mas/build-docs.mjs

```diff
@@ -81,7 +77,17 @@ const htmlTemplate = `
   <!-- Include your custom element script as an ES6 module -->
   <script src="../../../features/spectrum-web-components/dist/theme.js" type="module"></script>
   <script src="../../../features/spectrum-web-components/dist/button.js" type="module"></script>
-  ${masJs}
+  <script type="module" src="../../../deps/mas/mas.js"></script>
+  <mas-commerce-service></mas-commerce-service>
+  <script type="module">
+    const params = new URLSearchParams(document.location.search);
+    const masCommerceService = document.querySelector('mas-commerce-service');
+    ['locale','language','env'].forEach((attribute) => {
+      const value = params.get(attribute);
+      if (value) masCommerceService.setAttribute(attribute, value);
+    });
+    masCommerceService.activate();
```

Comment:
I think ` masCommerceService.activate();` should not be needed. As soon as it is attached to DOM, it should initialize. Ideally we should create ` <mas-commerce-service></mas-commerce-service>` with the set of attributes.